doc/source/whatsnew/v0.18.1.txt

```diff
@@ -105,6 +104,56 @@ API changes
 - ``pd.show_versions()`` now includes ``pandas_datareader`` version (:issue:`12740`)
 - Provide a proper ``__name__`` and ``__qualname__`` attributes for generic functions (:issue:`12021`)
 
+.. _whatsnew_0181.numpy_compatibility
+
+Compatibility between pandas array-like methods (e.g. ```sum`` and ``take``) and their analogous
```

Comment:
say ``numpy counterparts``, rather than analagous

dont need (``generally with *args, and **kwargs``).